yb-voyager/cmd/import.go

```diff
@@ -404,6 +413,7 @@ func validateBatchSizeFlag(numLinesInASplit int64) {
 		defaultBatchSize = DEFAULT_BATCH_SIZE_YUGABYTEDB
 	}
 
+	// TODO: we might want to lift this restriction for non-transactional COPY (depends on testing of --batch-size flag)
```

Comment:
good point. can you create a ticket for this, we can discuss this after this PR.

Reply:
its there - https://yugabyte.atlassian.net/browse/DB-15906